### PR TITLE
Remove Bower Dependency, optimize gulp, build optimize

### PR DESCRIPTION
**Pull request by spboyer**
- Removed bower dependency in favor of npm
- Switched out `rimraf` package for `del` see - https://github.com/robrich/gulp-rimraf/issues/14 
- fixed jshint issues in site.js
- added `gulp build` and `default` gulp task for clean code practices 
- modified project.json to call `gulp build`

### CODE REVIEW

File: src/live.asp.net/gulpfile.js [hunk 1]
```diff
@@ -56,4 +87,8 @@ gulp.task("csslint", function() {
     return gulp.src(paths.css)
         .pipe(csslint())
         .pipe(csslint.reporter());
-});
\ No newline at end of file
+});
+
+gulp.task("build", ["clean", "min", "lib", "csslint", "jshint"]);
```
Comment (DamianEdwards) [inline src/live.asp.net/gulpfile.js]: Doesn't this result in all the dependent tasks running in parallel?

File: src/live.asp.net/wwwroot/js/site.js [hunk 2]
```diff
@@ -1,10 +1,10 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved. 
+﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 (function (window, undefined) {
-    "use strict";
+    'use strict';
```
Comment (DamianEdwards) [inline src/live.asp.net/wwwroot/js/site.js]: no no no no no, I like `"` :wink:

Comment (spboyer) [inline src/live.asp.net/wwwroot/js/site.js]: violates the jshint rules, can change back and add ignore...

File: src/live.asp.net/gulpfile.js (hunk 1 above)
Comment (spboyer) [inline src/live.asp.net/gulpfile.js]: yes, made changes to new commit for all tasks to return streams and dependencies. missed that the clean tasks were not returning this. Nullifies the point of gulp really.

**Comment by DamianEdwards** [comment]
@spboyer this still appears to have some conflicts. Can you rebase and send again pls?

**Pull request closed without merging**